### DISCUSSION TITLE
I created another folder and content but it is not reflected on the Docs menu

### DISCUSSION THREAD

**Discussion by kkmb34** [thuliteio/doks#62] [Q&A] [answered]
Hi,

Sorry for the very noob question. I downloaded the theme and wanted to create another section to default Docs page. For this, I duplicated Help folder and created another folder called Help2. But It is not being reflected on the platform and it is still displayed as a green color with A symbol. ( A - Added (This is a new file that has been added to the repository))

I stopped the local server and started again but still not working.

![Capture6](https://user-images.githubusercontent.com/47612943/103389722-e34d8380-4b10-11eb-81c1-4af58842fa60.PNG)

By the way, I also added the following code to menus.toml file. So I can see the header on the side menu but it is empty. 

```
[[docs]]
  name = "Help2"
  weight = 70
  identifier = "help2"
  url = "/docs/help2/"
```
![Capture8](https://user-images.githubusercontent.com/47612943/103389855-9b7b2c00-4b11-11eb-975b-6aa0b2b2dccc.PNG)

**Comment by h-enk** [comment, marked as answer]
Hi @kkmb34, you're almost there...

Make sure that:
1. you are referencing the correct parent identifier (set in `menus.toml`) as the parent menu item — e.g. `help2`, and
2. the page title is unique — make it e.g. `FAQ2`

E.g. for `./content/docs/help2/faq.md`

![2020-12-31_11-01-17](https://user-images.githubusercontent.com/3902872/103405542-ca1bf580-4b57-11eb-9e8f-0617886e78c3.png)

Then you will get:

![2020-12-31_11-02-02](https://user-images.githubusercontent.com/3902872/103405784-955c6e00-4b58-11eb-8d92-0f4fc41c55e2.png)

**Reply by kkmb34** [reply]
thank you very much for your help. Now it works like a charm :)
